shardingsphere-ui/shardingsphere-ui-frontend/package.json

```diff
@@ -21,8 +21,8 @@
     "lodash": "^4.17.11",
     "moment": "^2.24.0",
     "normalize.css": "^8.0.1",
+    "resize-detector": "^0.2.1",
```

Comment:
What is this lib? can you add notice for that?

Reply:
This library is the one that the official Vue version of echarts relies on, so I also quoted